src/main/kotlin/com/wire/apps/polls/dto/common/Mapper.kt

```diff
@@ -0,0 +1,65 @@
+package com.wire.apps.polls.dto.common
+
+import com.wire.apps.polls.dto.Option
+import com.wire.apps.polls.dto.PollAction
+import com.wire.apps.polls.dto.UsersInput
+import com.wire.integrations.jvm.model.QualifiedId
+import com.wire.integrations.jvm.model.WireMessage
+import io.ktor.features.BadRequestException
+import java.util.UUID
+
+fun mapToUsersInput(wireMessage: WireMessage.Text): UsersInput? {
+    val sender = wireMessage.sender
+    sender ?: throw BadRequestException("Sender must be set for text messages.")
+    val conversationId = wireMessage.conversationId
+    val text = wireMessage.text ?: return null
```

Comment:
Okay we talked about why you returned the nullable UsersInput. Better not to pass nullable objects around to much but here there is probably an issue with the SDK.
@alexandreferris is there a reason why the text field inside WireMessage.Text is nullable ? By looking at the protobuf and the code I think we can make it not-nullable.

If that is the case @MarianKijewski keep the code like this and once we change in the SDK all this issue for you goes away and you simplify the code

Reply:
Seems like there is no apparent reason to have it nullable, I have changed it to non-nullable in my current implementation task and should be available in the next release.